src/Range.php

```diff
@@ -1,6 +1,8 @@
 <?php
 namespace IPTools;
 
+use ReturnTypeWillChange;
```

Comment:
What about this line? Does it cause a conflict with php 5?

Reply:
It does not.